src/schema.rs

```diff
@@ -47,6 +47,9 @@ pub struct Response {
     pub schema: Option<ReferenceOr<Schema>>,
     #[serde(default, skip_serializing_if = "IndexMap::is_empty")]
     pub headers: IndexMap<String, ReferenceOr<Header>>,
+
+    #[serde(default, rename = "x-ms-error-response", skip_serializing_if = "std::ops::Not::not")]
```

Comment:
This will work until someone adds a `x-ms-error-response": false` for some reason and then the roundtrip tests will fail. In #18, I switched all of them to `Option<bool>`.

Reply:
I'll update it now.